Corona-Warn-App/src/main/res/navigation/nav_graph.xml

```diff
@@ -96,7 +96,7 @@
     <fragment
         android:id="@+id/onboardingDeltaInteroperabilityFragment"
         android:name="de.rki.coronawarnapp.ui.onboarding.OnboardingDeltaInteroperabilityFragment"
-        android:label="OnboardingDeltaInteroperabilityFragment" >
+        android:label="OnboardingDeltaInteroperabilityFragment">
```

Comment:
Unrelated changes?

Reply:
was touched during commit history. I think its ok to mob up sometimes. Its clean reformat.

Reply:
why not be a path finder and leave every place you touch a little bit improved?